### PR TITLE
Fix UneditableFields

### PR DESCRIPTION
**Pull request by simonkern**
This PR resolves issue #1223 by restoring the layout of UneditableFields, as it was before 65897114358ab9d8b0da3f5d0b791d2ce77e63d0 and adds tests for it.

### CODE REVIEW

File: crispy_forms/tests/test_layout.py [hunk 1]
```diff
@@ -739,6 +739,33 @@ def test_form_control_size():
...
+    [
+        (True, "bootstrap4/test_layout/test_use_custom_control_in_uneditable_select_true.html"),
+        (False, "bootstrap4/test_layout/test_use_custom_control_in_uneditable_select_false.html"),
+    ],
+)
+def test_use_custom_control_in_uneditable_select(use_custom_control, expected_html):
+    form = SelectSampleForm()
+    form.helper = FormHelper()
+    form.helper.template_pack = "bootstrap4"
+    form.helper.layout = Layout("select")
```
Comment (smithdc1) [inline crispy_forms/tests/test_layout.py:764]: I think we should make this field an `UneditableField`?

**Review by smithdc1 — COMMENTED** [review]
Thanks for the PR. It looks great :+1:. Just one comment on the Select test, I think that needs a slight tweak to cover the changes to the template?

**Comment by simonkern** [comment]
my bad, it was too late yesterday.
fixed now

**Comment by smithdc1** [comment]
Thanks for the quick update. This looks great. :pray:

**Pull request merged**